io/cmd.go

```diff
@@ -83,13 +83,17 @@ func RunCmdWithOutputParser(config CmdConfig, prompt bool, regExpStruct ...*CmdO
 	if err != nil {
 		return
 	}
-	defer cmdReader.Close()
+	defer func() {
+		_ = cmdReader.Close()
```

Comment:
Let's not ignore the error returned from this function, and return it, if the function returns a nil error.
The same goes for line 95.